imports/components/@primitives/UI/live/index.js

```diff
@@ -79,14 +81,24 @@ class LiveWithoutData extends Component {
     const { live, show, embedCode } = this.props.live;
     if (!live || !show || !embedCode) return null;
 
+    // TODO load different embed code for beta users
+
+    // create beta link
+    const shouldShowBetaLink = (
+      this.props.person
+      && !this.props.person.authLoading
+      && this.props.person.authorized
+    );
+    const link = shouldShowBetaLink ? `/wowza/${embedCode}` : `/video/${embedCode}`;
+
     return (
       <Motion
```

Comment:
@jbaxleyiii any thoughts on testing this? Right now, [the snapshots](https://github.com/NewSpring/Holtzman/blob/master/imports/components/%40primitives/UI/live/__tests__/__snapshots__/index.js.snap#L23) don't actually render out the link, so I can't think of a way to check the redirect for beta testers. I messed with this for a while today, and I can't seem to mock `Motion` and get the function that is has as a child to call and render `Link`